include/TrustWalletCore/TWAnySigner.h

```diff
@@ -17,4 +17,7 @@ struct TWAnySigner;
 /// Signs a transaction.
 extern TWData *_Nonnull TWAnySignerSign(TWData *_Nonnull input, enum TWCoinType coin);
 
+/// Plan a transaction (for UTXO chains).
+extern TWData *_Nonnull TWAnySignerPlan(TWData *_Nonnull input, enum TWCoinType coin);
```

Comment:
A few things:
1. Swift / Java binding of TWAnySigner is not done by codegen (note no `TW_EXPORT`), so we have to add it manually
2. Looks like we can remove `UTXOPlanner` and keep this one